bluepyopt/ephys/evaluators.py

```diff
@@ -40,6 +40,7 @@ def __init__(
             fitness_protocols=None,
             fitness_calculator=None,
             isolate_protocols=None,
+            timeout=None,
```

Comment:
Could you move this argument to the end of the list. I'm always a bit afraid people who don't specify labels when calling the function might get in trouble when the order is not maintained